examples/gcp/main.tf

```diff
@@ -9,18 +9,18 @@ provider "google" {
 module "bitmovin_cloud_connect" {
   project_id = local.project_id
   source     = "../../modules/gcp" # local reference when you clone the repository
-  # source     = "github.com/bitmovin/terraform-cloud-connect/blob/main/modules/gcp" # remote GitHub reference
+  # source     = "github.com/bitmovin/terraform-cloud-connect//modules/gcp" # remote GitHub reference
 
   # For all possible input variables, please check:
   # Documentation: https://github.com/bitmovin/terraform-cloud-connect/blob/main/README.md#inputs
 }
 
 module "bitmovin_cloud_connect_with_specific_subnets" {
   project_id = local.project_id
-  source = "git@github.com:bitmovin/terraform-cloud-connect.git//modules/gcp"
+  source     = "github.com/bitmovin/terraform-cloud-connect//modules/gcp"
```

Comment:
@simontischler `"git@github.com:bitmovin/terraform-cloud-connect.git//modules/gcp"` did not work for me, but `"github.com/bitmovin/terraform-cloud-connect//modules/gcp"` did. So I just adapted this to use the same `source` path style.